azext_iot/central/_help.py

```diff
@@ -396,6 +397,28 @@ def _load_central_device_twin_help():
     """
 
 
+def _load_central_dps_keygen_help():
+    helps[
+        "iot central dps-keygen"
+    ] = """
+        type: group
+        short-summary: Generate a device SAS key
+    """
+
+    helps[
+        "iot central dps-keygen create"
+    ] = """
+        type: command
+        short-summary: Create a device SAS key
+        Long-summary: Generate device key from primary group SAS token. Generated device SAS key is used to connect the device to central.
```

Comment:
Might be worth mentioning a big use case for this functionality is offline key generation for factory scenarios